app/main/index.ts

```diff
@@ -341,21 +345,21 @@ app.on('ready', () => {
 		});
 	});
 
-	ipcMain.on('realm-name-changed', (event, serverURL, realmName) => {
+	ipcMain.on('realm-name-changed', (_event: Electron.IpcMessageEvent, serverURL: string, realmName: string) => {
 		page.send('update-realm-name', serverURL, realmName);
 	});
 
-	ipcMain.on('realm-icon-changed', (event, serverURL, iconURL) => {
+	ipcMain.on('realm-icon-changed', (_event: Electron.IpcMessageEvent, serverURL: string, iconURL: string) => {
 		page.send('update-realm-icon', serverURL, iconURL);
 	});
 
 	// Using event.sender.send instead of page.send here to
 	// make sure the value of errorReporting is sent only once on load.
-	ipcMain.on('error-reporting', event => {
+	ipcMain.on('error-reporting', (event: Electron.IpcMessageEvent) => {
```

Comment:
Shouldn't this be `_event` as well?

Reply:
This was a xo rule which we removed later on so _ is not required for unused parameters now.